package.json

```diff
@@ -28,9 +28,10 @@
     "standard": "^6.0.6"
   },
   "dependencies": {
-    "electron-prebuilt": "^1.2.2",
+    "electron": "^1.4.3",
```

Comment:
1.4.3 was just released and among other things added new methods to `NativeImage`, one of which is `resize` which I am planning to use for scaling higher resolution images to fit on pptx slides.  Hint: creating powerpoints will be an upcoming PR, and I will likely add dependencies on `officegen` and a TBD library to convert pdf pages into images.  The reasoning is that PDF export does a better job with handling page breaks than can be done by image capture alone.